docs/References.bib

```diff
@@ -448,6 +448,32 @@ @book{Cockburn1999
   url        = "https://doi.org/10.1007/978-3-662-03882-6_2"
 }
 
+@ARTICLE{Cook1992,
+  author =       {{Cook}, Gregory B. and {Shapiro}, Stuart L. and {Teukolsky},
+                  Saul A.},
+  title =        "{Spin-up of a Rapidly Rotating Star by Angular Momentum Loss:
+                  Effects of General Relativity}",
+  journal =      {The Astrophysical Journal},
+  year =         1992,
+  month =        oct,
+  volume =       {398},
+  pages =        {203},
+  doi =          {10.1086/171849},
+}
+
+@ARTICLE{Cook1994,
```

Comment:
these aren't generating links to the papers

Reply:
I don't know of any online verison of these papers